src/query/expression/src/schema.rs

```diff
@@ -98,12 +103,15 @@ pub static INTERNAL_COLUMNS: LazyLock<HashSet<&'static str>> = LazyLock::new(||
         ORIGIN_BLOCK_ID_COL_NAME,
         ORIGIN_BLOCK_ROW_NUM_COL_NAME,
         ROW_VERSION_COL_NAME,
+        FILENAME_COLUMN_NAME,
+        FILE_ROW_NUMBER_COLUMN_NAME,
     ])
 });
 
 #[inline]
 pub fn is_internal_column_id(column_id: ColumnId) -> bool {
     column_id >= SEARCH_SCORE_COLUMN_ID
+        || (FILE_ROW_NUMBER_COLUMN_ID..=FILENAME_COLUMN_ID).contains(&column_id)
```

Comment:
1. can we change the ID value of ORIGIN_xxx_COLUMN_IDs? 
2. `is_internal_column_id` should renamed to `is_computed_internal_column_id`

Reply:
stream ids can not be moved. we will move other ids in next pr. @zhyass